KomaMRIBase/src/sequences/RF/block_pulse.jl

```diff
@@ -0,0 +1,28 @@
+
+function block_pulse(flip_angle::FlipAngle, duration::Duration;
+    phase_offset=0, freq_offset=0, delay=0, sys=Scanner())
+
+    flip_angle, duration = flip_angle.val, duration.val
+    amplitude = flip_angle / (2π * γ * duration) * exp(im * phase_offset)
+    dead_time = sys.RF_dead_time_T
+    ring_down_time = sys.RF_ring_down_T
+    delay = max(delay, dead_time)
+    block_duration = delay + duration + ring_down_time
+
+    rf = RF(amplitude, duration, freq_offset, delay)
+    return Sequence([Grad(0, 0);;], [rf;;], [ADC(0, 0)], block_duration)
```

Comment:
we need a way to combine events so this is easier, `seq += [rf, block_duration]` where `rf::RF` and `block_duration::Delay`, issue #4.